qiling/extensions/r2/__init__.py

```diff
@@ -1 +1,39 @@
+from qiling import Qiling
 from .r2 import R2
+from .mem import R2Mem
+from .deflat import R2Deflator
+
+from unicorn.unicorn_const import UC_PROT_READ, UC_PROT_WRITE, UC_PROT_EXEC
+
+
```

Comment:
`__init__` should not contain funcionality logic directly, but only imports.
Better split the classes to different files, even if they are short.